stackslib/src/main.rs

```diff
@@ -974,6 +975,61 @@ simulating a miner.
         process::exit(1);
     }
 
+    if argv[1] == "get-tenure-inv" {
+        let chainstate_root_path = &argv[2];
+        let tip_block_ids = &argv[3..];
+        let chainstate_path = format!("{}/chainstate", &chainstate_root_path);
+        let sortition_path = format!("{}/burnchain/sortition", &chainstate_root_path);
+
+        let (chainstate, _) =
+            StacksChainState::open(false, 0x80000000, &chainstate_path, None).unwrap();
+        let pox_consts =
+            PoxConstants::new(900, 100, 80, 0, 0, u64::MAX, u64::MAX, 240, 241, 242, 242);
```

Comment:
This is a lot of magic numbers

Reply:
This is a one-off tool that I wrote to test this against the live nakamoto testnet. I'm fine with just dropping the patch to this file.